src/timer/timer_enable.cc

```diff
@@ -75,7 +81,7 @@ class TimerStatCollector {
 
     void Report(std::size_t index, cr_duration_nsec_t duration);
 
-    void Merge(const TimerStatCollector& collector);
+    void Merge(const TimerStatCollector& another);
```

Comment:
I believe it's a lot more common to say "other" rather than "another".

Reply:
Don't know that we even have a convention here :)

We are merging with *another* (single) collector instead of (multiple) *other* collectors, that is why I choose "another". I don't think it is relevant, though.

Reply:
> Don't know that we even have a convention here :)

No we don't, I'm only saying it's quite often to find people using `other` for copy ctor in their libs, while I can't really recall seeing "another" anywhere.
Plus it's shorter.

> I don't think it is relevant, though.

That's true, but this is a common use case, for unnamed copy/move ctor arg, so would be nice to have a good template/suggestion for people to copy around, rather than name each one on their own.